### PR TITLE
Move 2678 to Last Call

### PR DESCRIPTION
**Pull request by njgheorghita**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2678 state from Review to Last Call

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
SPDX should refer to https://spdx.org/licenses/ not Wikipedia.

----

We should see how other editors feel about this, but I'm not a fan of HTML being used in EIPs for formatting.  It conflates data and presentation, and we shouldn't assume that all places where this EIP will be rendered will support embedded HTML nor should editors/maintainers have to deal with the responsiveness issues brought about by HTML/CSS.

Is there a compelling reason we cannot just use regular markdown tables here?

**Comment by njgheorghita** [comment]
@MicahZoltu I've updated the SPDX link and converted the html tables to markdown in #3284 . There's still a little bit of html left in the document though - mostly to define the anchor points for various in-page links. As far as I'm aware there's no markdown mechanism to replicate this functionality.

**Comment by MicahZoltu** [comment]
Ping me on this PR once #3284 is merged.  Also, see my note about Markdown anchor links in there.

**Comment by njgheorghita** [comment]
@MicahZoltu Thoughts about the status of this pr? #3284 and the other formatting prs have been merged. Is this ready to merge? Does the `Last Call` date need updating to start 2 weeks from now?

File: EIPS/eip-2678.md
Comment (MicahZoltu) [inline EIPS/eip-2678.md:7]:
```suggestion
review-period-end: 2021-03-23
```

**Comment by MicahZoltu** [comment]
Unfortunately we do need to set the review end date to 2 weeks from merge, so people have enough time to give feedback once notifications go out.  I have gone ahead and done this, will merge as soon as CI lets me.

**Comment by njgheorghita** [comment]
Thanks @MicahZoltu !

**Pull request merged**
